# Transcription
DNA is wrapped around histone octomers tightly enough to inhibit accessibility by RNA polymerase. How does transcription start? One process is the acetylation or methylation of the histone octomer, which will electrostatically push the DNA away from the histones, making it accessible. And then there is:

## Nucleosome Remodeling Complex ("NRC")
The NRC changes how the DNA is wrapped around the octomer so it can be accessed by RNA polymerase. NRCs are ATP-driven, because what they do is highly energetic: they can move, eject, or restructure nucleosomes.
### Nucleosome Movement
#### Sliding
NRCs can "shift" a nucleosome up or down the strand to make more room between histome octomers, like rolling a spool of thread so that it travels along the strand, while the parts of the thread not on the spool don't change.
This can expose transcription factor binding sites (TFBS) for a gene, which would allow RNA polymerase to get in and start working.
#### Ejecting
NRCs can knock the histone octomer out of the doubly coiled DNA entirely, like loosening a thread around a spool enough you can pull the spool off. Can have the same effect as above.
#### Restructuring
NRCs can modify specific histones to be a different protein, which changes the electrostatics around that part of DNA. E.g. H3 can turn into H3.3, and H2A can turn into H2AX. This is one way that the repair machinery can get in to fix a break in the DNA strand.
### NRC types
All the NRC types have an ATPase domain; they need ATP to operate given how energetic their work is.
| NRC Type | # subunits | Bromo/Chromodomain | Slide? | Eject? | Restructure? |
| -------- | ---------- | ------------------ | ------ | ------ | ------------ |
| SWI/SWF | 8-11 | Bromo | Y | Y | Y |
| ISWI | 2-4 | No | Y | N | N |
| Mi2/Chd | 8-10 | Chromo | Y | N | N |
#### Bromo/Chromodomains
These are protein domains that are involved in gene regulation. They bind to specific histone modifications: bromodomains bind to acetylated lysine residues, chromodomains typically bind to methylated lysine residues. Their interactions help recruit other proteins to specific locations in DNA, which can influenec gene expression.

ISWI
* organizes nucleosomes into their proper form
* plays a central role in chromatin assembly after DNA replication
* maintains higher-order chromatin structure

NuRD/Mi2/Chd
* mediates transcriptional reoression
* is required for the pluripotency of embryonic stem cells
## Transcription factors and transcription
In eukaryotic cells, there are three kinds of RNA polymerases that each make different kinds of RNA. They all needs another protein to get into the DNA to start making RNA, which is called a general transcription factor.
### Transcription factor I
TF I makes rRNA (ribosomal), which goes to the ribosome, then is translated.
### Transcription factor II
TF II makes mRNA. mRNA is modified (splicing), then translated. This process also makes small nuclear RNAs (snRNA), which are involved in the process of splicing.
### Transcription factor III
TF III makes tRNA (transfer), some snRNA, and a little rRNA.

The process of changing the speed of transcription of a gene is called "gene regulation", and has its own file.
